File: Carro.py
class Carro:
    def __init__(self, marca, modelo, ano, cor, preco):
        self.__marca = marca
        self.__modelo = modelo
        self.__ano = ano
        self.__cor = cor
        self.__preco = preco

    def get_marca(self):
        return self.__marca

    def set_marca(self, nova_marca):
        self.__marca = nova_marca

    def get_modelo(self):
        return self.__modelo

    def set_modelo(self, novo_modelo):
        self.__modelo = novo_modelo

    def get_ano(self):
        return self.__ano

    def set_ano(self, novo_ano):
        self.__ano = novo_ano

    def get_cor(self):
        return self.__cor

    def set_cor(self, nova_cor):
        self.__cor = nova_cor

    def get_preco(self):
        return self.__preco

    def set_preco(self, novo_preco):
        self.__preco = novo_preco


# Exemplo de uso da classe Carro
meu_carro = Carro("Toyota", "Corolla", 2020, "Preto", 50000)

# Obtendo os valores dos atributos
print("Marca:", meu_carro.get_marca())
print("Modelo:", meu_carro.get_modelo())
print("Ano:", meu_carro.get_ano())
print("Cor:", meu_carro.get_cor())
print("Preço:", meu_carro.get_preco())

# Alterando valores dos atributos
meu_carro.set_modelo("Camry")
meu_carro.set_ano(2021)
meu_carro.set_cor("Branco")
meu_carro.set_preco(55000)

# Verificando as alterações
print("Novo modelo:", meu_carro.get_modelo())
print("Novo ano:", meu_carro.get_ano())
print("Nova cor:", meu_carro.get_cor())
print("Novo preço:", meu_carro.get_preco())

File: Carro.java
public class Carro {
    private String marca;
    private String modelo;
    private int ano;
    private String cor;
    private double preco;

    public Carro(String marca, String modelo, int ano, String cor, double preco) {
        this.marca = marca;
        this.modelo = modelo;
        this.ano = ano;
        this.cor = cor;
        this.preco = preco;
    }

    public String getMarca() {
        return marca;
    }

    public void setMarca(String novaMarca) {
        this.marca = novaMarca;
    }

    public String getModelo() {
        return modelo;
    }

    public void setModelo(String novoModelo) {
        this.modelo = novoModelo;
    }

    public int getAno() {
        return ano;
    }

    public void setAno(int novoAno) {
        this.ano = novoAno;
    }

    public String getCor() {
        return cor;
    }

    public void setCor(String novaCor) {
        this.cor = novaCor;
    }

    public double getPreco() {
        return preco;
    }

    public void setPreco(double novoPreco) {
        this.preco = novoPreco;
    }

    public static void main(String[] args) {
        Carro meuCarro = new Carro("Toyota", "Corolla", 2020, "Preto", 50000);

        // Obtendo os valores dos atributos
        System.out.println("Marca: " + meuCarro.getMarca());
        System.out.println("Modelo: " + meuCarro.getModelo());
        System.out.println("Ano: " + meuCarro.getAno());
        System.out.println("Cor: " + meuCarro.getCor());
        System.out.println("Preço: " + meuCarro.getPreco());

        // Alterando valores dos atributos
        meuCarro.setModelo("Camry");
        meuCarro.setAno(2021);
        meuCarro.setCor("Branco");
        meuCarro.setPreco(55000);

        // Verificando as alterações
        System.out.println("Novo modelo: " + meuCarro.getModelo());
        System.out.println("Novo ano: " + meuCarro.getAno());
        System.out.println("Nova cor: " + meuCarro.getCor());
        System.out.println("Novo preço: " + meuCarro.getPreco());
    }
}
